src/influxdb_ioxd/http.rs

```diff
@@ -785,6 +809,94 @@ async fn snapshot_partition<M: ConnectionManager + Send + Sync + Debug + 'static
     }
 }
 
+#[tracing::instrument(level = "debug")]
+async fn pprof_home<M: ConnectionManager + Send + Sync + Debug + 'static>(
+    req: Request<Body>,
+) -> Result<Response<Body>, ApplicationError> {
+    let default_host = HeaderValue::from_static("localhost");
+    let host = req
+        .headers()
+        .get("host")
+        .unwrap_or(&default_host)
+        .to_str()
+        .unwrap_or_default();
+    let cmd = "/pprof/profile?seconds=1";
```

Comment:
```suggestion
    let cmd = "/pprof/profile?seconds=30";
```


In Go land it's normal for a CPU `profile` to default to `30 seconds`. I think we should probably do the same

Reply:
makes sense; done

Reply:
avoided duplicating the default value here; instead using `PProfArgs::default_seconds()`